tablo/geom_utils.py

```diff
@@ -437,4 +437,196 @@ def extract_significant_digits(number):
     if is_negative:
         rounded = 0 - rounded
 
-    return rounded
\ No newline at end of file
+    return rounded
+
+
+def wkt_to_esri_feature(wkt):
+    geom_type = wkt[:wkt.find('(')]
+
+    def _geom_repl():
+        bracket_multiplier = 1
+        if geom_type.find('LINESTRING') > -1:
+            bracket_multiplier = 2
+        return json.loads(re.sub(
+            WKT_GEOM_2D_REGEX,
+            lambda m: '[{}]'.format(m.group().replace(' ', ',')),
+            wkt.replace(geom_type, '')
+        ).replace('(', '[' * bracket_multiplier).replace(')', ']' * bracket_multiplier))
+
+    if geom_type == 'POINT' or geom_type == 'MULTIPOINT':
+        match = WKT_POINT_REGEX.match(wkt)
+        if not match:
+            raise ValueError('Invalid Point Geometry: {0}'.format(wkt))
+        x, y = (float(x) for x in match.groups())
```

Comment:
Also keep in mind that in `tablo.exceptions` we have some `ValidationError` handling defined... without being immersed in it at this point I can't recommend anything specifically, and I see that we are using `ValueError` throughout the file, but I thought it might be worth mentioning.